src/PKSim.CLI/app.config

```diff
@@ -7,7 +7,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="SX" publicKeyToken="882b9c044052e7f6" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-2.7.6.1" newVersion="2.7.6.1" />
+        <bindingRedirect oldVersion="0.0.0.0-2.7.8.1" newVersion="2.7.8.1" />
```

Comment:
@Yuri05 Can you replace your app.config with the version of v9.1. That should be enough